libfwupdplugin/fu-device.h

```diff
@@ -447,6 +447,15 @@ typedef guint64 FuDeviceInternalFlags;
  */
 #define FU_DEVICE_INTERNAL_FLAG_MD_SET_SIGNED (1ull << 23)
 
+/**
+ * FU_DEVICE_INTERNAL_FLAG_FORCE_ENUMERATION:
+ *
+ * Forces composite device components to be enumerated.
+ *
+ * Since: 1.7.7
+ */
+#define FU_DEVICE_INTERNAL_FLAG_FORCE_ENUMERATION (1ull << 24)
```

Comment:
is this something we are going to use outside the tbt plugin?

Reply:
probably not

Reply:
fu_device_register_private_flag to the rescue